patches/@react-native-segmented-control+segmented-control+2.4.0.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/node_modules/@react-native-segmented-control/segmented-control/index.d.ts b/node_modules/@react-native-segmented-control/segmented-control/index.d.ts
```

Comment:
Why did we need this patch?

Reply:
Sounds good! I'll open an issue to undo this when we bump for the next Expo release